pkg/probe/server.go

```diff
@@ -33,6 +33,11 @@ func (h *Handler) Wait(interrupt chan os.Signal) error {
 
 			for i := range h.waitProbes {
 				err := h.waitProbes[i].Exec()
+				var pathErr *os.PathError
+				if errors.As(err, &pathErr) {
+					log.WithFields(log.Fields{"kind": "probe", "name": i, "err": err}).Fatal("Path does not exist!")
+					return nil
```

Comment:
To keep log messages consistent with the rest of the repo, i'd suggest we log in lower-case here (and without special characters), too. Also, as `logrus.Fatal` will exit the program, any code below it won't be reached.
```suggestion
					log.WithFields(log.Fields{"kind": "probe", "name": i, "err": err}).Fatal("path does not exist")
```

Reply:
`logrus.Fatal `did not break the loop unfortunatly. Thats why I added `return nil `